addons/website/views/snippets/s_product_catalog.xml

```diff
@@ -70,11 +70,10 @@
                 <!-- Note: no need of extra dependency thanks to the apply-to -->
             </t>
         </div>
-        <div data-selector=".s_product_catalog_dish"/>
+        <div data-selector=".s_product_catalog_dish" data-drop-near=".s_product_catalog_dish"/>
     </xpath>
-    <xpath expr="//div[@data-js='SnippetMove']" position="attributes">
+    <xpath expr="//div[@data-js='SnippetMove'][contains(@data-selector,'section')]" position="attributes">
```

Comment:
Missing space after the `,` but in fact I don't think that change is needed at all?

Reply:
without this fix, any snippet can be dropped into the product catalog snippet...
IMO we dont want it

Reply:
oh you mean just this line

Reply:
It's my bad for copy pasting from timeline without actually understanding what this was used for. It was indeed useless so I removed it. Sorry 😅